README.md

```diff
@@ -12,6 +12,24 @@ Oracle enhanced ActiveRecord adapter provides Oracle database access from Ruby o
 
 INSTALLATION
 ------------
+### Rails 5 - in branch rails5
+
+Rails 5 is still in pre release, so use with caution.
```

Comment:
Rails 5 itself has been released, how about "Oracle enhanced adapter Rails 5 support is ..."?